doctr/models/detection/fast/pytorch.py

```diff
@@ -279,6 +279,47 @@ def ohem_sample(score: torch.Tensor, gt: torch.Tensor, mask: torch.Tensor) -> to
         return text_loss + kernel_loss
 
 
+def reparameterize(model: Union[FAST, nn.Module]) -> FAST:
```

Comment:
I don't understand what is the purpose of reparameterization.
First, we need to instantiate the model, then to modify some layers. Is it for inference purpose ?

Reply:
yep in simple words it replaces all FASTConvLayers with simplier Conv2D layers, merges the weights and set init the Conv2D with it 

reparameterization is only onces applied before inference :)